lib/trento/infrastructure/alerting/emails/templates/api_key_expiration.html.eex

```diff
@@ -0,0 +1,426 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <title>Simple Transactional Email</title>
+  <style>
```

Comment:
We need to study if this styles can be imported from other file, as we use them in the other template as well.
Could we open a tech debt ticket?

Reply:
Yes, they are standard eex file, we can template and have components to import, I will open a tech debt ticket

Reply:
Created